pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -54,19 +57,19 @@ public class JsonAsyncHttpPinotClientTransport implements PinotClientTransport {
   private final Map<String, String> _headers;
   private final String _scheme;
 
-  private static final long BROKER_READ_TIMEOUT_MS = 60000L;
-  private static final int BROKER_CONNECT_TIMEOUT_MS = 2000;
-
+  private final int _brokerReadTimeout;
   private final AsyncHttpClient _httpClient;
 
   public JsonAsyncHttpPinotClientTransport() {
+    _brokerReadTimeout = 60000;
     _headers = new HashMap<>();
     _scheme = CommonConstants.HTTP_PROTOCOL;
     _httpClient = Dsl.asyncHttpClient();
   }
 
   public JsonAsyncHttpPinotClientTransport(Map<String, String> headers, String scheme,
-    @Nullable SSLContext sslContext) {
+    @Nullable SSLContext sslContext, int readTimeout, int connectTimeout, int handshakeTimeout, boolean tlsV10Enabled) {
```

Comment:
instead of passing a boolean flag for tls 1.0, why not pass a list of protocols?

Reply:
I could make it a list, but it might be a bit complicated to pass that as a JDBC string argument. It might also introduce possible misconfiguration. A list is more flexible, so I could try to parse it from the connection string. Let me know what you prefer.

Reply:
I introduced two parameter objects